### ISSUE TITLE
Having Problems When Sharing Multiple Files to Phone

### ISSUE THREAD

**Issue by Colin-Y0470** [nearby-sharing/android#184]
Every time I choose two or more files to share from Windows to my phone, it will fail and inform you "invalid operation" on my phone.

**Comment by ShortDevelopment** [comment]
@Colin-Y0470 Could you send me your logs please (See [logging](https://nearshare.shortdev.de/technical/logging/))?

**Comment by Colin-Y0470** [comment]
Yes, here are some information maybe helpful for you. Sorry, one image is in Chinese.
![屏幕截图 2024-11-11 191655](https://github.com/user-attachments/assets/da56aa58-0ce0-4c85-923e-b2b4b8f2e820)
![S41111-19142429_de shortdev nearby_sharing_windows(1)](https://github.com/user-attachments/assets/ec5d5e08-4ebd-434d-9e6d-6541abd0ba64)
[nearshare-android-20241016.log.txt](https://github.com/user-attachments/files/17701714/nearshare-android-20241016.log.txt)
[nearshare-android-20241111.log.txt](https://github.com/user-attachments/files/17701716/nearshare-android-20241111.log.txt)

**Comment by A3148475824** [comment]
The computer transfers the files to the mobile phone, and there is no notification on the Nearby Sharing software on the mobile phone!

**Comment by ShortDevelopment** [comment]
@Colin-Y0470 Your second log seems to be caused by #188.

This should be fixed in the next release.
If its not: Please notify me here.
